resources/assets/js/components/AcceptPayment.vue

```diff
@@ -69,6 +69,13 @@
                 <li>Follow the prompts to complete the transaction.</li>
                 <li>It is not required to provide a paper receipt for swiped card transactions.</li>
               </ol>
+            </template>
+              <template v-else-if="payment.method == 'square'">
+              <p class="font-weight-bold">Only use this payment method if you were specifically told to do so.</p>
+              <ul>
+                  <li>Square transactions incur a $3 processing fee. Add $3 to payment amount.</li>
+                  <li>This is <em>not</em> the same as SquareCash.</li>
```

Comment:
Not a fan of this copy. If this item is specifically being used only to clean up other system errors, then it should state that in the copy.

Reply:
This would only be displayed to the treasurer/other admins but I agree it should be rephrased.